<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliceWidget</class>
 <widget class="QWidget" name="SliceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SliceWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QSplitter" name="sph_settingAndViewer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QScrollArea" name="sa_imageviewer">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>169</width>
         <height>320</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <spacer name="vspr_setting">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="gb_sliceSetting">
          <property name="title">
           <string>Slice Setting</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QLineEdit" name="le_nozzleSize"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lb_layerHeight">
             <property name="text">
              <string>Layer Height (mm)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lb_nozzleSize">
             <property name="text">
              <string>Nozzel Size (mm)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="le_layerHeight"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="gb_machineSetting">
          <property name="title">
           <string>Machine Setting</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="lb_temperature">
             <property name="text">
              <string>Temperature</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="le_temperature"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lb_filamentDiameter">
             <property name="text">
              <string>Filament Diameter</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="le_filamentDiameter"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="lb_feedRate">
             <property name="text">
              <string>Feed Rate (mm/s)</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLineEdit" name="le_feedRate"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QScrollArea" name="sa_viewer">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>298</width>
         <height>320</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSlider" name="vslid_image">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="btmbox_widget">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
